src/main/java/com/odysseusinc/arachne/portal/service/submission/impl/BaseSubmissionServiceImpl.java

```diff
@@ -159,6 +157,8 @@
 
     @Value("${files.store.path}")
     private String fileStorePath;
+    @Value("${portal.explicitExecutionApprove}")
```

Comment:
Why do you add props which are never used? If it is required for enterprise, add it there